tests/test_monitoring.py

```
@@ -0,0 +1,520 @@
+# ---------------------------------------------------------------------------
+
+
+class TestStartupConfigValidation:
+    """Tests for validate_startup_config."""
+
+    def test_no_warnings_with_good_config(self):
+        env = {
+            "AI_PROVIDER": "claude",
+            "AI_MODEL": "test-model",
+            "JJI_ENCRYPTION_KEY": "test-key",
+            "DB_PATH": "/tmp/test.db",
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/test.db"

(S108)